src/dispatchers/keyDispatcher.ts

```diff
@@ -1,12 +1,13 @@
 ///<reference path="../reference.ts" />
 
 module Plottable {
-export module Dispatchers {
-  export type KeyCallback = (keyCode: number, event: KeyboardEvent) => void;
+export type KeyCallback = (keyCode?: number, event?: KeyboardEvent) => void;
```

Comment:
We should move this `KeyCallback` back to being under `Plottable.Dispatchers.KeyCallback`.

Otherwise this constitutes an API break.